### DISCUSSION TITLE
Vim 9 support - any estimate of when that will happen?

### DISCUSSION THREAD

**Discussion by bobharrisbtt** [macvim-dev/macvim#1250] [General]
I know Vim 9 was just released, and I realize it will take some time to incorporate the update into MacVim.

I'm just wondering if there is an estimate of when that might happen.

**Comment by eirnym** [comment]
Hi @bobharrisbtt, thank you for a question. It's projected to be soon enough. 

Please, follow the issue https://github.com/macvim-dev/macvim/issues/1235 be aware of what's going on.

**Reply by ychin** [reply]
#1235 is now done.
